macros/set_warehouse_size.sql

```diff
@@ -8,23 +8,26 @@
 
 {% macro snowflake__set_warehouse_size(size) %}
 
-	{% if var("available_sizes", None) == None %}
-		{{ exceptions.raise_compiler_error("Please set the `available_sizes` variable in the dbt_project.yml.") }}
+	{% if var("snowflake_warehouses", None) == None %}
+		{{ exceptions.raise_compiler_error("Please set the `snowflake_warehouses` variable in the dbt_project.yml.") }}
```

Comment:
NB: It would be nice to not require setting a config on a new dbt project, and only rely on the warehouse set by the `profiles.yml`. But your call, you have more context and are actually using it :D